### ISSUE TITLE
Depth Norm

### ISSUE THREAD

**Issue by Golbstein** [ialhashim/DenseDepth#36]
```
def DepthNorm(x, maxDepth):
    return maxDepth / x
```

Shouldn't it be `x/maxDepth`?

**Comment by ialhashim** [comment]
The goal of this normalization is to have the network give higher loss on areas closer to the camera. In the NYU dataset, nearby objects are much more accurate (due to the hardware used to acquire the data). If you set `maxDepth` to 1000 (e.g. 10 meters) then the closest possible values to the camera would be `1000/10` and the furthest will be `1000/1000` (notice we clip values to 10 to avoid dividing by zero). Now when the model is training the loss is much higher if you don't learn to estimate the nearby points too well.

You are free to try and experiment with any normalization or loss function. In practice this was giving me good results.
